clients/filter/filter.go

```diff
@@ -82,7 +86,7 @@ func (c *Client) Healthcheck() (string, error) {
 }
 
 // GetOutput returns a filter output job for a given filter output id
-func (c *Client) GetOutput(ctx context.Context, filterOutputID string) (m Model, err error) {
+func (c *Client) GetOutput(ctx context.Context, filterOutputID string, serviceAuthToken string, downloadServiceToken string) (m Model, err error) {
```

Comment:
I personally dislike the _named return parameters_. I think they are misleading and can lead to var shadowing which can be a royal pain to workout if you have a defect.
I appreciate this is not your change but I'd prefer to update this and have implicit return values.